README.md

```diff
@@ -17,7 +17,7 @@
  * under the License.
 -->
 
-weinre is WEb INspector REmote.
+weinre is Web INspector REmote.
```

Comment:
The capitalization is there for a reason:

we = WEb
in = INspector
re = REmote
=> weinre

Reply:
Oh, thanks for that.